hack/testdata/values-v1.7.0.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for navigator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+createAPIService: true
+
+rbac:
+  enabled: true
+
+apiserver:
+  ## Set to true to skip deploying the apiserver components RBAC policies,
+  ## which require cluster admin access to deploy.
+  rbacDisabled: false
+
+  extraArgs:
+  - --requestheader-client-ca-file=/var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+  - --requestheader-username-headers=X-Remote-User
+  - --requestheader-group-headers=X-Remote-Group
+  - --requestheader-extra-headers-prefix=X-Remote-Extra
+  - --v=100
+
+  ## Optional: if not set, a service account will be automatically created
+  # serviceAccount: "apiserver-svc-acct"
+  image:
+    repository: jetstackexperimental/navigator-apiserver
+    tag: latest
+    pullPolicy: Never
+
+controller:
+  ## Optional: namespace to watch for resources in. This can be used when RBAC
+  ## restricts you to a single namespace.
+  # namespace: default
+  ## Optional: if not set, a service account will be automatically created
+  # serviceAccount: "controller-svc-acct"
+  image:
+    repository: jetstackexperimental/navigator-controller
+    tag: latest
+    pullPolicy: Never
+
+resources:
+  requests:
+    cpu: 50m
+    memory: 64Mi
```

Comment:
`helm install` says `  -f, --values valueFiles      specify values in a YAML file (can specify multiple) (default [])` so perhaps we could put only the `apiserver.extraArgs` in here?

Reply:
Yep we could - I'm inclined to keep this nice and explicit though, so it's easy to understand what's being deployed in the e2e tests.